### ISSUE TITLE
MIgrating CmuSltHsmmvoice built in MaryTTS 4.3.0 to voices built in AndroidMaryTTS

### ISSUE THREAD

**Issue by kantajit** [AndroidMaryTTS/AndroidMaryTTS#4]
I have the working model of voice CmuSltHsmm that was built using MaryTTS 4.3.0. I want to import that voice model to AndroidMaryTTS.

I noticed there are some changes in .pdf files and in HTSCARTReader.java .
Actually when I tried for Cmu-slt-hsmm, I noticed there are some changes in .pdf files and in HTSCARTReader.java.

So I ended up doing some changes in HTSCARTReader.java.
After these changes mary engine started up ok, but I am getting wrong Audio. 

Is there any way to migrate voice model that was built using MaryTTS 4.3.0 to voice compatible to AndroidMaryTTS?

**Comment by farizaghayev** [comment]
Yes of course compatible, We nothing have changed in voice generated java files.  Share with us your voice model and languages model. we can find whats problem or fork this repo and add your files
